resumebuilder/routes.py

```diff
@@ -43,4 +44,31 @@ def login():
 @app.route('/logout')
 def logout():
     logout_user()
-    return redirect(url_for('index'))
\ No newline at end of file
+    return redirect(url_for('index'))
+
+
```

Comment:
include @login_required decorator from flask_login to prevent page being accessed by unauthenticated users.